sdk/internal/ci.yml

```diff
@@ -3,12 +3,14 @@ trigger:
   paths:
     include:
     - sdk/internal/
+    - eng/
```

Comment:
Why does this need to be added?

Reply:
I'm doing this in a separate PR so I think this was a mental block adding it here, it will get added eventually so I'm going to keep it as is.